spinnaker/modules/vault-agent/vault.yaml

```diff
@@ -0,0 +1,33 @@
+global:
+  enabled: true
+
+injector:
+  enabled: true
+
+  image:
+    repository: "hashicorp/vault-k8s"
+    tag: "0.3.0"
```

Comment:
This is the `vault-k8s` version, not the chart version.

Reply:
is there a reason to tag the *vault image* instead of using the one supplied with the helm chart already?
Upgrades to the helm chart(like the one to 0.4.0) might prove to be problematic if we pin the *vault image* version

Reply:
It's better practice to version is it not?

Reply:
i take it back. I was right the first time. the vault-k8s just supplies the injector.

Reply:
if you version it and the rest of the helm chart updates including the version of the vault container, but not the agent injector, i suspect there will be greater problems than upgrading the helm chart and bringing along the vault version, the vault k8s integration and the other things all together.